code/game/gamemodes/gang/gang.dm

```diff
@@ -247,6 +248,20 @@ var/list/gang_colors_pool = list("red","orange","yellow","green","blue","purple"
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	if(forced_shuttle)
+		return
+	var/normies = 0 
+	for(var/mob/living/M in GLOB.player_list)
+		if(isliving(M) && !M.stat==DEAD && (!M.mind in get_all_gangsters()))
```

Comment:
There's more wrong with this, !M.stat, !M.mind, and calling get_all_gangsters every iteration

Change the whole bit to:
```
var/list/all_gangsters = get_all_gangsters()
for(var/mob/living/L in GLOB.player_list)
    if(M.stat != DEAD && !(L.mind in all_gangsters))
```